<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Loan project" duration-ms="206" started-at="2018-08-16T17:38:25Z" finished-at="2018-08-16T17:38:25Z">
    <groups>
    </groups>
    <test name="Car loan" duration-ms="206" started-at="2018-08-16T17:38:25Z" finished-at="2018-08-16T17:38:25Z">
      <class name="test.parameters">
        <test-method status="PASS" signature="WebsiteCarLoan(java.lang.String, java.lang.String)[pri:0, instance:test.parameters@1efea79]" name="WebsiteCarLoan" duration-ms="15" started-at="2018-08-16T17:38:25Z" finished-at="2018-08-16T17:38:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[paraloan.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[12345]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- WebsiteCarLoan -->
        <test-method status="PASS" signature="users(java.lang.String, java.lang.String)[pri:0, instance:test.parameters@1efea79]" name="users" duration-ms="1" started-at="2018-08-16T17:38:25Z" data-provider="getData" finished-at="2018-08-16T17:38:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[first username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- users -->
        <test-method status="PASS" signature="users(java.lang.String, java.lang.String)[pri:0, instance:test.parameters@1efea79]" name="users" duration-ms="2" started-at="2018-08-16T17:38:25Z" data-provider="getData" finished-at="2018-08-16T17:38:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[second username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- users -->
        <test-method status="PASS" signature="users(java.lang.String, java.lang.String)[pri:0, instance:test.parameters@1efea79]" name="users" duration-ms="3" started-at="2018-08-16T17:38:25Z" data-provider="getData" finished-at="2018-08-16T17:38:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[third username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- users -->
        <test-method status="FAIL" signature="users1()[pri:0, instance:test.parameters@1efea79]" name="users1" duration-ms="104" started-at="2018-08-16T17:38:25Z" finished-at="2018-08-16T17:38:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at test.parameters.users1(parameters.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- users1 -->
      </class> <!-- test.parameters -->
    </test> <!-- Car loan -->
  </suite> <!-- Loan project -->
</testng-results>
